store/README.md

```diff
@@ -0,0 +1,167 @@
+Get a list of proofs for given nullifier hashes, each proof as Tiered Sparse Merkle Trees ([read more](../proto/proto/tsmt.proto)).
+
+**Parameters:**
+
+* `nullifiers`: `[Digest]` – array of nullifier hashes.
+
+**Returns:**
+
+* `proofs`: `[NullifierProof]` – array of nullifier proofs, positions correspond to the ones in request.
+
+### GetBlockHeaderByNumber
+
+Retrieves block header by given block number.
+
+**Parameters**
+
+* `block_num`: `uint32` *(optional)* – the block number of the target block. If not provided, the latest known block will be returned.
+
+**Returns:**
+
+* `block_header`: `BlockHeader` – block header.
+
+### GetBlockInputs
+
+Returns data needed by the block producer to construct and prove the next block.
+
+**Parameters**
+
+* `account_ids`: `[AccountId]` – array of account IDs. 
+* `nullifiers`: `[Digest]` – array of nullifier hashes (not currently in use).
+
+**Returns**
+
+* `block_header`: `[BlockHeader]` – the latest block header.
+* `mmr_peaks`: `[Digest]` – peaks of the above block's mmr, The `forest` value is equal to the block number.
+* `account_states`: `[AccountBlockInputRecord]` – the hashes of the requested accouts and their authentication paths.
+* `nullifiers`: `[NullifierBlockInputRecord]` – the requested nullifiers and their authentication paths.
+
+### GetTransactionInputs
+
+Returns account and nullifiers descriptors. 
+
+**Parameters**
+
+* `account_id`: `AccountId` – account ID.
+* `nullifiers`: `[Digest]` – array of nullifier hashes.
+
+**Returns**
+
+* `account_state`: `AccountTransactionInputRecord` – account's descriptors. 
+* `nullifiers`: `[NullifierTransactionInputRecord]` – the requested nullifiers' blocks at which ones have been consumed, zero if not consumed.
+
+### SyncState
+
+State synchronization request.
+
+**Parameters**
+
+* `block_num`: `uint32` – send updates to the client starting at this block.
+* `account_ids`: `[AccountId]`
+* `note_tags`: `[uint32]` – note tags filter. Corresponds to the high 16 bits of the real values, shifted right (`value >> 48`).
+* `nullifiers`: `[uint32]` – nullifiers filter. Corresponds to the high 16 bits of the real values, shifted right (`value >> 48`).
+
+**Returns**
+
+* `chain_tip`: `uint32` – number of the latest block in the chain.
+* `block_header`: `BlockHeader` – block header of the block with the first note matching the specified criteria.
+* `mmr_delta`: `MmrDelta` – data needed to update the partial MMR from `block_ref` to `block_header.block_num`.
+* `block_path`: `MerklePath` – Merkle path in the updated chain MMR to the block at `block_header.block_num`.
+* `accounts`: `[AccountHashUpdate]` – a list of account hashes updated after `block_ref` but not after `block_header.block_num`.
+* `notes`: `[NoteSyncRecord]` – a list of all notes together with the Merkle paths from `block_header.note_root`.
+* `nullifiers`: `[NullifierUpdate]` – a list of nullifiers created between `block_ref` and `block_header.block_num`.
+
+### ListNullifiers
```

Comment:
The 3 "List*" endpoints are meant for testing purposes only - so, I'd probably put them under a separate sub-header.